lightkurve/lightcurve.py

```diff
@@ -1213,22 +1213,37 @@ def _create_plot(self, method='plot', ax=None, normalize=False,
                 xlabel = 'Time'
         # Default ylabel
         if ylabel is None:
-            if normalize or (self.flux.unit == u.dimensionless_unscaled):
-                ylabel = 'Normalized Flux'
-            elif self.flux.unit is None:
-                ylabel = 'Flux'
+            if "flux" in column:
+                    ylabel = "Flux"
             else:
-                ylabel = 'Flux [{}]'.format(self.flux.unit.to_string("latex_inline"))
+                    ylabel = f"{column}"
+            if normalize or self.meta.get("normalized"):
+                ylabel = "Normalized " + ylabel
+            elif (self[column].unit) and (self[column].unit.to_string() is not ''):
```

Comment:
@barentsen I added the `(self[column].unit.to_string() is not '')` check for the following reason: I wanted to continue having the unit printed in square brackets for consistency, but wanted to avoid an empty "[]" appearing for the unitless columns. Even though these columns have no units, their data still appear in astropy Quantities, and so always passed this if statement.